src/vs/platform/menubar/electron-main/menubar.ts

```diff
@@ -36,6 +36,9 @@ export class Menubar {
 	private closedLastWindow: boolean;
 
 	private menuUpdater: RunOnceScheduler;
+	private menuGC: RunOnceScheduler;
+
+	private oldMenus: Menu[];
```

Comment:
@sbatten this needs a comment at least why we are doing it with a reference to the electron issue and a TODO to remove this when Electron fixed it!